packages/PowerGrid/README.md

```diff
@@ -5,14 +5,50 @@ and support for [images](#images).
 
 ## Sparklines
 
-Sparklines are a very small charts, typically without axes or coordinates. The main goal of a sparkline is 
-to provide a graphical representation of a data object that allows not only to get an impression of the object 
-as a whole but also visually instantly compare objects with each other. To form a representation, any quantitative 
-features of objects can be used, which are displayed as the position or size of a graphical element.
-Datagrok has a few ready-to-use sparkline chart types 'sparklines' 'bar chart' 'pie chart' and 'radar chart'. 
-You can customize which columns / objects' features will be used to form the graph.
-
-![](../../help/develop/how-to/custom-cell-renderers-sparklines-and-settings.gif "Sparklines and settings")
+Sparklines are small charts that are built based on columns with a countable data. They help users to analyze and compare objects in data sets quickly by presenting them in a visual way. In Datagrok sparklines are available in four different reprentations: sparklines, bar charts, pie charts, and radar charts. 
+
+Sparklines come in handy when you want to have a brief look at your data set and form a first impression. Lets check the example:
```

Comment:
> Sparklines come in handy when you want to have a brief look at your data set and form a first impression.

Isn't it pretty much the same as was written in the first sentence? "Sparklines are small charts that are built based on columns with a countable data". Let's combine the sentences as neither of them currently give a good description of what sparklines are.

Reply:
Current structure:

1) We tell what it is
2) What it does
3) Types
4) How is it useful
5) Example
6) How to use them

There are no "same" sentences.